readme.md

```diff
@@ -105,15 +105,15 @@ You can use the components by referencing the loader directly (via unpkg), or in
 ### Use via mgt-loader:
 
 ```html
-<script src="https://unpkg.com/@microsoft/mgt@3/dist/bundle/mgt-loader.js"></script>
+<script src="https://unpkg.com/@microsoft/mgt-components@3/dist/bundle/mgt-loader.js"></script>
```

Comment:
mgt-loader is not in the components package

Reply:
So CDN is only mgt-related?

Reply:
no, the mgt-loader.js file is only in one of our packages, trying to load it from the component package results in a 404.

This file is the only case, IMO, where we should be referencing the `@microsoft/mgt` package directly in our docs